### ISSUE TITLE
Nothing spawns when using 0.9.3

### ISSUE THREAD

**Issue by andymwat** [BoredBean/TouhouCompany#5]
None of the models or sounds are replaced when using 0.9.3. The only log messages related to TouhouEnemyModels that are logged are:
``` 
[Info   :   BepInEx] Loading [TouhouEnemyModels 0.9.3]
[Info   :TouhouEnemyModels] Plugin TouhouEnemyModels is loaded!
```

[LogOutput.log](https://github.com/BoredBean/TouhouCompany/files/15053237/LogOutput.log)

**Comment by BoredBean** [comment]
The whole Start() function has been jumped.

Please provide a list of all the mods you enabled. A share code would be fine.

**Comment by andymwat** [comment]
Tested on 018f05ea-64a8-1395-49f7-b827888c0318

Only mod besides the enemy models is Imperium, debug mod used to spawn them in. A friend tested without it just to make sure though.

**Comment by BoredBean** [comment]
BepInEx/config/BepInEx.cfg

```
[Chainloader]

## If enabled, hides BepInEx Manager GameObject from Unity.
## This can fix loading issues in some games that attempt to prevent BepInEx from being loaded.
## Use this only if you know what this option means, as it can affect functionality of some older plugins.
## 
# Setting type: Boolean
# Default value: false
HideManagerGameObject = false
```

When this is set to false, the start function doesn't work. 
I'm still looking for more clues.

**Comment by BoredBean** [comment]
One week past.
I tried to ask the community but no response.
So, I will make the dependency to LCSoundTool weak.
We don't really need it in the next version.

**Comment by BoredBean** [comment]
Released at tag v0.9.5.
